src/solvers/solvers_parabolic.jl

```diff
@@ -28,4 +28,6 @@ struct ViscousFormulationLocalDG{P}
     penalty_parameter::P
 end
 
+ViscousFormulationLocalDG() = ViscousFormulationLocalDG(nothing)
```

Comment:
I guess the docstring above has to be updated to mention the new capabilities and also the paper you referenced in your post?